README.rst

```diff
@@ -60,6 +60,14 @@ You can run only the boto3 tests with::
 
 This section contains some basic tests for the AssumeRole, GetSessionToken and AssumeRoleWithWebIdentity API's. The test file is located under ``s3tests_boto3/functional``.
 
+To run the STS tests, the vstart cluster should ne started with the following parameter (in addition to any parameters already used with it)::
+
+        vstart.sh -o rgw_sts_key=abcdefghijklmnop -o rgw_s3_auth_use_sts=true
```

Comment:
actually rgw_sts_key can be set to anything that is 128 bits in length.

Reply:
but it does work with the above, right?
any reason to change that?

Reply:
it works, we could just mention that the key given is an example or something similar.

Reply:
@yuvalif : can you just add here that the key used is an example.

Reply:
added. see line 59

Reply:
ok